include/ppx/fs.h

```diff
@@ -143,6 +143,13 @@ std::filesystem::path GetInternalDataPath();
 std::filesystem::path GetExternalDataPath();
 #endif
 
+// Returns the default output directory where test artifacts will be written.
```

Comment:
nit: maybe we should remove the word `test`? I think testing is one use case for outputting files, but I don't think it's the only one

Reply:
Good point, I removed it.